services/ui-src/src/routes/index.tsx

```diff
@@ -7,5 +7,7 @@ export { NotFound } from "./NotFound/NotFound";
 export { Profile } from "./Profile/Profile";
 // report
 export { GetStarted } from "./GetStarted/GetStarted";
-export { Dashboard } from "./Dashboard/Dashboard";
+export { DashboardPage } from "./Dashboard/DashboardPage";
+export { DashboardList } from "./Dashboard/DashboardList";
+export { MobileDashboardList } from "./Dashboard/MobileDashboardList";
```

Comment:
I know we talked about moving these from `/routes` to `/components/pages` -- maybe this is a good time to start moving some of those over? Or maybe not. It's 100% fine here for now. Just a thought.